src/modules/app/containers/App.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 
-import * as React from 'react'
+import React from 'react'
```

Comment:
I think we have to stick to `* as React` because of flow. I think something like `React.Node` does not work without.